sdk/storage/storage-blob/src/generated/src/models/parameters.ts

```diff
@@ -1254,10 +1254,10 @@ export const snapshot: coreHttp.OperationQueryParameter = {
     }
   }
 };
-export const sourceContentCrc64: coreHttp.OperationParameter = {
+export const sourceContentcrc64: coreHttp.OperationParameter = {
```

Comment:
`sourceContentcrc64` or sourceContentCrc64`?

Reply:
Which one, @bterlson ?

Reply:
@tg-msft

Reply:
We rename this to `sourceContentHash` with a transform in .NET so we don't have quite the same name pop-up.  I think `sourceContentCrc64` is what you want.  We should add a transform for now.

Reply:
Definitely sourceContentCrc64. The other one feels like a typo somewhere to me.

Reply:
Done